index.html

```diff
@@ -0,0 +1,524 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://kit.fontawesome.com/ea35fd0321.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>TNW</title>
+</head>
+<body>
+  <main>
+    <header>
+      <nav>
+        <div class="logo-header">
+          <img src="images/logo.png" alt="logo">
+        </div>
+        <div class="upper-menus">
+          <ul class="top-bar1">
+            <li><a href="#">News</a></li>
+            <li><a href="#">Events</a></li>
+            <li><a href="#">Business</a></li>
+            <li><a href="#">AMAs</a></li>
+            <li><a href="#">Spaces</a></li>
+            <li><a href="#">Terms & Conditions</a></li>
+          </ul>
+          <ul class="top-bar2">
+            <li><a href="#">About</a></li>
+            <li><a href="#">Team</a></li>
+            <li><a href="#">Advertise</a></li>
+            <li><a href="#">Jobs</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </div>
+        <div class="lower-menus">
+          <ul class="low-bar">
+            <li><a href="#">LATEST</a></li>
+            <li><a href="#">HARD FORK</a></li>
+            <li><a href="#">PLUGGED</a></li>
+            <li><a href="#">FUNDAMENTALS</a></li>
+            <li><a href="#">WORK 2030</a></li>
+          </ul>
+          <button class="serch-button">
+            <i class="fas fa-search fa-2x"></i>
+          </button>
+          <button class="hidden-menu">
+            <i class="fas fa-bars fa-2x"></i>
+          </button>
+        </div>
+        <div class="nav-message">
+          <a href="#">Did you get your tickets to Hard Fork Summit yet? →</a>
+        </div>
+      </nav>
+    </header>
+    <div class="main-articles">
+        <div class="article one">
+          <header>
+            <div class="cover-logo">
+              <img src="images/tnwlogo.png" alt="">
+            </div>
+          </header>
```

Comment:
The `<header>` tag is used for the header of the page not the divs, please remove the unnecessary header tags through the content.